### ISSUE TITLE
DCP405: triggers unwanted Standby

### ISSUE THREAD

**Issue by fusedFET** [eez-open/modular-psu-firmware#298]
When disabling a channel, the BB3 sometimes goes into standby.
Firmware: 1.7.3
HW: DCP405 R2B11

There is a large part of random to reproduce it, but I now have encountered it multiple times.

[Video](https://1drv.ms/v/s!AvQwX7vex5MhpsEIU7Si8z3VcWPu3w?e=f1olbi)

**Comment by fusedFET** [comment]
Here is the event log when having the issue:
![image](https://user-images.githubusercontent.com/8060929/144858654-c91f6a89-647c-495a-907d-500ee85c59bf.png)

**Comment by prasimix** [comment]
Ok, so you probably have this option enabled: 
![image](https://user-images.githubusercontent.com/15106476/144865843-766fe07a-8ffb-4e56-93d9-217c5a350ffb.png)

**Comment by fusedFET** [comment]
Correct, I have this setting enabled.

**Comment by prasimix** [comment]
That's fine, that is the trip which bring them into standby mode due to that option activated. So it seems that some overshoot is present on the output enabled. Is it possible that you have load connected with longer cables on that channel?

**Comment by fusedFET** [comment]
I do have some longer cables, about 40 cm.
Maybe in order to clarify the issue, it would be practical to have a message on screen when going into standby for this reason:
"OVP Triggered Standby..." ?

**Comment by prasimix** [comment]
> I do have some longer cables, about 40 cm. Maybe in order to clarify the issue, it would be practical to have a message on screen when going into standby for this reason: "OVP Triggered Standby..." ?

Yes, that makes sense, otherwise it can look confusing (and scary) if the user forgets that he chose the option to go to standby because of the trip.
